kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit_ja.properties

```diff
@@ -36,6 +36,7 @@ label.ok = OK
 label.cancel = 取消
 label.close = 閉じる
 label.copy = コピー
+label.save.as=\u540D\u524D\u3092\u4ED8\u3051\u3066\u4FDD\u5B58
```

Comment:
Can we add Japanese Text instead of Unicode?

Reply:
Yes, it seems to work. For some reason this didnt work the first time.